widget/embedded/src/pages/SelectBlockchainPage.tsx

```diff
@@ -24,15 +25,17 @@ export function SelectBlockchainPage(props: PropTypes) {
   const [blockchainCategory, setBlockchainCategory] = useState<string>('ALL');
   const setToBlockchain = useQuoteStore.use.setToBlockchain();
   const setFromBlockchain = useQuoteStore.use.setFromBlockchain();
-  const fetchStatus = useAppStore().fetchStatus;
-
+  const { fetchStatus, setSelectedBlockchainForCustomToken } = useAppStore();
   const blockchains = useAppStore().blockchains({
     type,
   });
 
   const activeCategoriesCount = getCategoriesCount(blockchains);
 
-  const showCategory = activeCategoriesCount !== 1;
+  const showCategory =
+    props.showCategory === false
```

Comment:
You can change `showCategory` to `hideCategory` in your props. and update this code to something like this:
```
const showCategory = props.hideCategory : false : activeCategoriesCount !== 1;
```